### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by evelinabis**
About webrick: I had problems starting the server like in https://github.com/jekyll/jekyll/issues/8523. I found a solution there.

### CODE REVIEW

File: docs/reports_dest.html [hunk 1]
```diff
@@ -1 +1 @@
-http://api.ugene.net/stats/savestats.php
\ No newline at end of file
+https://api.ugene.net/stats/savestats.php
```
Comment (yalgaer) [inline docs/reports_dest.html]: This should not be changed: this is the way UGENE reports data to the server.
If we change it to HTTPS we need to be sure that our QT app can safely connect to our HTTPS.

Same for all our server side API endpoints: their certificate can expire (we had such problems before) or use too an incompatible certificate (too old one).

File: docs/Gemfile [hunk 2]
```diff
@@ -27,4 +27,5 @@ end
 
 # Performance-booster for watching directories on Windows
 gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
+gem "webrick"
```
Comment (yalgaer) [inline docs/Gemfile:30]: Why do we need this?

**Review by yalgaer — CHANGES_REQUESTED** [review]
Please remove HTTPS endpoints for API calls.
Everything else looks good.

File: docs/Gemfile (hunk 2 above)
Comment (evelinabis) [reply docs/Gemfile:30]: As I mentioned in [the comment](https://github.com/ugeneunipro/ugeneunipro.github.io/pull/62#issue-1217087533), I had problems starting the server. Did you have any problems with this when you checkout to the tr branch?

This is an optional change, I can remove it.

File: docs/reports_dest.html (hunk 1 above)
Comment (evelinabis) [reply docs/reports_dest.html]: Fair enough, corrected.

File: docs/Gemfile (hunk 2 above)
Comment (yalgaer) [reply docs/Gemfile:30]: No I never had this problem. Lets keep it if it helps you to run the site.

**Review by yalgaer — APPROVED** [review]

**Comment by yalgaer** [comment]
Please check the site after merging. It will auto-deploy

**Pull request merged**